src/cli_util.rs

```diff
@@ -903,7 +903,8 @@ impl WorkspaceCommandHelper {
         {
             ui.write_warn(r#"Name and email not configured. Add something like the following to $HOME/.jjconfig.toml:
   user.name = "Some One"
-  user.email = "someone@example.com""#)?;
+  user.email = "someone@example.com"
+"#)?;
```

Comment:
I suspect you'll have to update `test_global_opts.rs`, but CI will tell you in that case. It might be that something about in the snapshot-testing framework adds a final newline because it looks like the test already has it.

Reply:
Yeah, I tried running the tests and it didn't seem to notice that there was a newline or not. I think the snapshot-testing framework is just ignoring it for testing convenience, perhaps?

Reply:
Yes, that's the only thing I can think of. I don't care enough to look into it anyway :)

Reply:
@martinvonz that snapshot framework is not sensitive to trailing newlines. I forget exactly why, but it was worse when it *was*. I think it had something to do with Rust's multi-line string literal rules.

Reply:
Ah, I can see something like that getting annoying. Thanks for explaining.